skl2onnx/operator_converters/linear_classifier.py

```diff
@@ -53,11 +59,11 @@ def convert_sklearn_linear_classifier(scope, operator, container):
                 is not SOFTMAX.
                 https://github.com/scikit-learn/scikit-learn/blob/bac89c253b35a8f1a3827389fbee0f5bebcbc985/sklearn/linear_model/base.py#L300
                 """ # noqa
-                classifier_attrs['post_transform'] = 'LOGISTIC'
+                classifier_attrs['post_transform'] = post_transform
             else:
-                classifier_attrs['post_transform'] = 'LOGISTIC'
+                classifier_attrs['post_transform'] = post_transform
         else:
-            classifier_attrs['post_transform'] = 'LOGISTIC'
+            classifier_attrs['post_transform'] = post_transform
     elif op.__class__.__name__ in ('LinearSVC'):
```

Comment:
This is not needed as we already checked for LinearSVC above.

Reply:
Can you fix this?

Reply:
I think I did.

Reply:
I don't see any difference between lines 47-48 and 55-56 in this file.

Reply:
My mistake. Fixed.